src/main/java/com/emc/pravega/perf/PerfStats.java

```diff
@@ -110,24 +113,21 @@ public synchronized void printAll() {
      */
     public synchronized void printTotal(Instant endTime) {
         final long elapsed = Duration.between(start, endTime).toMillis();
-        double recsPerSec = 1000.0 * iteration / (double) elapsed;
+        double recsPerSec = 1000.0 * count / (double) elapsed;
```

Comment:
This multiplying by `1000.0` could be moved onto elapsed, then it wouldn't need to be duplicated. The same pattern is in `printWindow()` above.

Reply:
Thanks.. yes, this done now.. elapsed will be in seconds now.